processor/attributewhitelistprocessor/testData/testConfig.yaml

```
@@ -0,0 +1,18 @@
+receivers:
+  examplereceiver:
+
+exporters:
+  exampleexporter:
+
+processors:
+  attributewhitelist:
+    event_white_list:
+      - \bsomething\b
+      - http.*
```

should we add anchors and escape the period in this one? that's what we'd probably want in production
```suggestion
      - ^http\..*
```

lets update the test for this also